src/status_im2/contexts/chat/messages/list/style.cljs

```diff
@@ -0,0 +1,83 @@
+(ns status-im2.contexts.chat.messages.list.style
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.platform :as platform]
+            [react-native.reanimated :as reanimated]))
+
+(defonce ^:const cover-height 168)
+(defonce ^:const overscroll-cover-height 2000)
+(defonce ^:const cover-bg-color "#2A799B33")
+(defonce ^:const header-avatar-top-offset -36)
+
+(def footer
+  {:z-index 5})
+
+(defn keyboard-avoiding-container
+  [view-height keyboard-height]
+  {:position :absolute
+   :display  :flex
```

Comment:
I believe `:flex` is the default value for the `:display` property, so including this is unnecessary.